uaclient/config.py

```diff
@@ -247,12 +247,14 @@ def _attached_service_status(
         self, ent, inapplicable_resources
     ) -> "Dict[str, str]":
         details = ""
+        description_override = ""
```

Comment:
Would the "no override" case be better represented by `None`?

Reply:
None and done ;)